### ISSUE TITLE
How to erase the code

### ISSUE THREAD

**Issue by alexisriva** [raycoarana/material-code-input#9]
I want to erase the code, but only if there's an error, and i can't find how to do that, please help.

**Comment by bcmfr** [comment]
Hello
just set this : 

`
mdp =  findViewById(R.id.codeViewInput);
mdp.addOnCompleteListener(new OnCodeCompleteListener() {
            @Override
            public void onCompleted(String code) {
                if (!code.equals("000000")){
                    mdp.setEditable(true);
                    mdp.setCode("");
                    //Show error
                    mdp.setError("Erreur de code!");
                }
                else{
                    mdp.setShowKeyboard(false);
                }
            }
        });`
basically mdp.setcode(""); just reset your code

**Comment by raycoarana** [comment]
As @bcmfr says, just set the code empty string and the code will be reset.
